syntheticsclientv2/get_checksv2.go

```diff
@@ -46,11 +48,27 @@ func (c Client) GetChecksV2(params *GetChecksV2Options) (*ChecksV2Response, *Req
 	if params.PerPage == 0 {
 		params.PerPage = int(50)
 	}
+	if params.SchedulingStrategy == "" {
+		params.SchedulingStrategy = ""
+	}
 
 	// Make the request
 	details, err := c.makePublicAPICall(
 		"GET",
-		fmt.Sprintf("/tests?testType=%s&page=%d&perPage=%d&orderBy=%s&search=%s", params.TestType, params.Page, params.PerPage, params.OrderBy, params.Search),
+		fmt.Sprintf("/tests?testType=%s&page=%d&perPage=%d&orderBy=%s&search=%s%s&schedulingStrategy=%s%s%s%s%s%s",
```

Comment:
Mind updating the json payload to include the fields you've added in this test?
https://github.com/splunk/syntheticsclient/blob/v2/syntheticsclientv2/get_checksv2_test.go#L30

Reply:
is this JSON payload real or just an example? its pretty out of date but i didn't want to mess anything up when updating

Reply:
Likely it is pretty old! Ideally it would match reality!
That said, as long as it works at least it works.

If you don't want to / plan on updating that this PR, please create an issue for this and can circle back and get it updated (maybe along with others) later.
Thanks!

Reply:
sounds good. i updated the example. turns out it wasn't too outdated, it was just missing `customProperties`, `lastRunStatus`, `lastRunAt` and needed to change the keys to camelCase.